### PR TITLE
add unit tests for CDC environmental data import

### PR DESCRIPTION
**Pull request by padma-g**
@pradh @spiekos I finally put these together!

### CODE REVIEW

File: scripts/us_cdc/environmental_health_toxicology/README.md [hunk 1]
```diff
@@ -50,24 +50,24 @@ The dataset documentation is accessible on the [CDC National Evnvironment Public
 ### Artifacts
 
 #### Scripts
-[`parse_air_quality.py`](data/scripts/us_cdc/environmental_health_toxicology/)
+[`parse_air_quality.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/scripts/parse_air_quality.py)
 
-[`parse_precipitation_index.py`](data/scripts/us_cdc/environmental_health_toxicology/)
+[`parse_precipitation_index.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py)
```
Comment (spiekos) [inline scripts/us_cdc/environmental_health_toxicology/README.md:56]: Also add a subsection that notes and links to the files needed for the test scripts and the expected output.

File: scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py [hunk 2]
```diff
@@ -1,64 +0,0 @@
-'''
-Author: Padma Gundapaneni @padma-g
```
Comment (spiekos) [inline scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py:2]: fix the header to match the other files including the Google copywrite and having more information on what the file does and what the inputs are

**Review by spiekos — CHANGES_REQUESTED** [review]

File: scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py (hunk 2 above)
Comment (padma-g) [reply scripts/us_cdc/environmental_health_toxicology/scripts/parse_precipitation_index.py:2]: I actually moved this script. The new file with the same name has this information!

File: scripts/us_cdc/environmental_health_toxicology/README.md (hunk 1 above)
Comment (padma-g) [reply scripts/us_cdc/environmental_health_toxicology/README.md:56]: done

File: scripts/us_cdc/environmental_health_toxicology/README.md [hunk 3]
```diff
@@ -50,24 +50,24 @@ The dataset documentation is accessible on the [CDC National Evnvironment Public
 ### Artifacts
 
 #### Scripts
```
[the remark this answers is not in the document]
Comment (padma-g) [reply scripts/us_cdc/environmental_health_toxicology/README.md:52]: done, added to the end of the README

**Review by spiekos — APPROVED** [review]

**Pull request merged**